packages/core/src/generators/move/generator.ts

```diff
@@ -148,7 +213,33 @@ function updateXmlReferences(tree: Tree, options: NormalizedSchema) {
         contents
           .toString()
           .replaceAll(pathToUpdate, newPath)
-          .replaceAll(pathToUpdateWithWindowsSeparators, newPath),
+          .replaceAll(pathToUpdateWithWindowsSeparators, newPath)
+          .replaceAll(options.currentNamespace, options.destinationNamespace),
+      );
+
+      updateReferencingDotnetCode(tree, projectDirectory, options);
+    }
+  });
+}
+
+function updateReferencingDotnetCode(
+  tree: Tree,
+  projectDirectory: string,
+  options: NormalizedSchema,
+) {
+  visitNotIgnoredFiles(tree, projectDirectory, (path) => {
+    const extension = extname(path);
+    if (['.cs', '.vb', '.fs'].includes(extension)) {
+      const contents = tree.read(path);
+      if (!contents) {
+        return;
+      }
+
+      tree.write(
+        path,
+        contents
+          .toString()
+          .replaceAll(options.currentNamespace, options.destinationNamespace),
```

Comment:
It may be worth having more advanced pattern matching here. As is, if your renaming a namespace called `Shop` and have a variable named something like `ShopCreator` it would update that variable declaration and may not be desired.

For now, this is **_likely_** fine, so don't worry about trying to improve it on this PR, just something to keep in mind.

Reply:
Good point,  I had my npm scoped deeply nested namespaces in my brain.  I agree better pattern matching would be safest.